src/types/character/character.ts

```diff
@@ -1,18 +1,18 @@
-import { Document } from "mongoose";
-import { IItem } from "../shop/item";
-import { IEffect } from "../skills/effect";
-import { IStat } from "./stat";
-
-export interface ICharacter extends Document {
-  credits: number;
-  effects: IEffect[];
-  experience: number;
-  gender: string;
-  inventory: IItem[];
-  inventoryMax: number;
-  inventorySpace: number;
-  level: number;
-  name: string;
-  ownerId: string;
-  stats: IStat[];
-}
+import { Document } from "mongoose";
+import { IItem } from "../shop/item";
+import { IEffect } from "../skills/effect";
+import { IStat } from "./stat";
+
+export interface ICharacter extends Document {
```

Comment:
You said that extends Document was useless

Reply:
And I also said something like: "I will change everywhere else in another PR" ^^